releasenotes/notes/remove-obfuscations-metric-c082f6fd63085c67.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: Remove the datadog.trace_agent.obfuscations count which can slow
```

Comment:
We don't need a change log on this PR. Just add the `no-changelog` label. This isn't something users care about or use. If it were, it'd have been more of a challenge to remove it.